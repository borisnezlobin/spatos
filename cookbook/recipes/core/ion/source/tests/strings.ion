echo hello?world
echo hello*world
echo hello^world
echo hello%world
echo "\$hello"
echo "\\"
echo "\n"
echo -e "one\\\\\ntwo\\\\\nthree"
echo "\""
echo "<"
echo '<'
echo "<<"
echo '<<'
echo "<<<"
echo '<<<'
echo ">"
echo '>'
echo ">>"
echo '>>'
echo ">>>"
echo '>>>'
