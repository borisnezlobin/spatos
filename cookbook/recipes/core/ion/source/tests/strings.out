hello?world
hello*world
hello^world
hello%world
$hello
\
\n
one\
two\
three
"
<
<
<<
<<
<<<
<<<
>
>
>>
>>
>>>
>>>
